lib/src/inline_parser.dart

```diff
@@ -128,10 +128,14 @@ class InlineParser {
     }
   }
 
+  /// Add [node] to the last [TagState] on the stack.
   void addNode(Node node) {
     _stack.last.children.add(node);
   }
 
+  /// Push [state] onto the stack of [TagState]s.
+  void pushTagState(TagState state) => _stack.add(state);
```

Comment:
Consider calling this `openTag()` to make it describe the semantics of what's happening (a new tag has been opened that will need to be closed eventually) rather than the mechanics.

Reply:
i like it. done.